tasks/post-deploy/08-enable-new-navbar.sh

```diff
@@ -1,6 +1,31 @@
 #!/bin/bash
 
-# Update new navbar design on dev sites
+# Update new navbar design on all dev sites
 if [ "$APP_ENV" = "development" ]; then
   wp option patch insert planet4_options new_design_country_selector 'on'
+  exit
 fi
+
+# All code below only runs on production/staging
+
+# Catch NROs on non-gp.org domains
+# ch, ummah, history, handbook, storytelling, gcefca
+if [[ -n "${APP_HOSTPATH}" ]]; then
+  echo "Excluded NRO: ${APP_HOSTNAME}"
+  exit
+fi
+
+# Other NRO websites with customized navbar
+excluded=(hongkong korea taiwan argentina japan)
+
+nro="${APP_HOSTPATH}"
+echo "NRO: ${nro}"
+
+for i in "${excluded[@]}"; do
+  if [ "$i" == "$nro" ]; then
+    echo "Excluded NRO"
+    exit
+  fi
+done
+
+wp option patch insert planet4_options new_design_country_selector 'on'
```

Comment:
There are individual options for country selector and navbar, this one only does country selector, you also need
```
wp option patch insert planet4_options new_design_navigation_bar 'on'
```
(same on l.5)

Reply:
I made the addition in both places.